src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,42 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
also you can use new syntax
```suggestion
  constructor(public name, public health = 100) {

```

Reply:
It doesn't work. Error: Parameter modifiers can only be used in TypeScript files.